generate/src/include/maat/config.hpp.in

```diff
@@ -82,8 +82,10 @@ public:
      * order:
      *     1. Explicit files added with 'add_explicit_sleigh_file'
      *     2. Explicit directories added with 'add_explicit_sleigh_dir'
-     *     3. Maat install directory, specified with `MAAT_INSTALL_DIR` env variable
-     *     4. Standard locations such as '/usr/local/', '/usr/', ...
+     *     3. Sleigh files directory, specified with `MAAT_SLEIGH_DIR` env variable
+     *     4. Maat install directory, specified with `MAAT_INSTALL_DIR` env variable
```

Comment:
Having `MAAT_SLEIGH_DIR` is a bit redundant with `MAAT_INSTALL_DIR` for a user installation, but it's useful for running the python tests without installing the spec files anywhere